src/frontend/rest.go

```diff
@@ -63,6 +63,8 @@ func buildURL(host, resource, id string) (string, error) {
 	return uri.String(), nil
 }
 
+// sendRestRequest abstracts sending GET and POST requests with a timeout, parsing response and error handling.
```

Comment:
nit: no need to repeat the method name in the comment (here and throughout)

Reply:
I followed the Go Doc pattern for public methods. Maybe it is excessive. 😞